cms/static/js/views/utils/xblock_utils.js

```diff
@@ -73,13 +73,23 @@ define(["jquery", "underscore", "gettext", "common/js/components/utils/view_util
         deleteXBlock = function(xblockInfo, xblockType) {
             var deletion = $.Deferred(),
                 url = ModuleUtils.getUpdateUrl(xblockInfo.id),
-                xblockType = xblockType || gettext('component');
+                additionalHeaderText = '',
+                additionalBodyText = '';
+            xblockType = xblockType || gettext('component');
+            if (xblockInfo.get('is_prereq')) {
+                additionalHeaderText = gettext(' (and prerequisite)');
+                additionalBodyText = gettext(
+                    ' Any content that has listed this content as a prerequisite ' +
```

Comment:
Don't break up strings in JavaScript that will be translated into multiple lines-- I'm pretty sure the i18n scraping fails on this.

Note that you should test your UI with mock translation done to verify that strings are getting scraped properly. See "Coverage testing" here: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html.